v2/test/ball.test.js

```diff
@@ -0,0 +1,12 @@
+import { Ball } from '../ball.js'
+
+test('Unit test with mocks', () => { 
```

Comment:
Maybe this test description could be refined, it looks like it's not just "unit test with mocks", but verifies something else with the status of ball object

Reply:
Ok! I am going to refine the test description. The function return a value set in the constructor of class ball.

Reply:
Thanks